### PR TITLE
add wait and clear form

### PR DESCRIPTION
**Pull request by lyie28**

### CODE REVIEW

**Comment by lyie28** [comment]
@adeprez 

I propose the following changes:

1) Fix the wait method which was leading to error because it was using exec_code with code (not our new action json structure)
2) Add a clear method to clear text in forms to handle cases where forms are incorrectly filled in from previous attempts and need to be cleared
3) Fix user_agent being added to Playwright - this was causing initializing to fail

**Comment by adeprez** [comment]
2. "Add a clear method to clear text in forms to handle cases where forms are incorrectly filled in from previous attempts and need to be cleared - the alternative I thought of would be to clear forms by default when inputting new information, but I wasn't sure if that might lead to some unwanted behavior in some cases where we delete something we wanted to keep!"

I wouldn't clear the whole form (some fields could be prefilled with correct values), but instead update the navigation engine to replace the value. So far I see no case where the World Model would like to append text to an input, it looks okay to systematically clear the value before we start typing.

File: lavague-integrations/drivers/lavague-drivers-selenium/lavague/drivers/selenium/base.py [hunk 1]
```diff
@@ -282,6 +290,16 @@ def get_capability(self) -> str:
   - xpath (string)
   - value (string)
 
+Name: wait
+Description: Wait for the amount of seconds specified as duration
+Arguments:
+  -  duration (float)
+
+Name: click
```
Comment (adeprez) [inline lavague-integrations/drivers/lavague-drivers-selenium/lavague/drivers/selenium/base.py:298]: Duplicate click action in the template

Comment (lyie28) [reply lavague-integrations/drivers/lavague-drivers-selenium/lavague/drivers/selenium/base.py]: Updated with latest commit :)

**Pull request merged**